tests/vm/centos.aarch64

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python
+#
+# Centos aarch64 image
+#
+# Copyright 2020 Linaro
+#
+# Authors:
+#  Robert Foley <robert.foley@linaro.org>
+#  Originally based on ubuntu.aarch64
+#
+# This code is licensed under the GPL version 2 or later.  See
```

Comment:
Is this license correct or you derived it from other file?

Reply:
QEMU is GPL 2 for the most part.  I copied this from one of the existing script files.  All the existing scripts here have the same notice.